doc/make.bat

```diff
@@ -35,6 +35,10 @@ goto end
 :clean
 rmdir /s /q %BUILDDIR% > /NUL 2>&1
 rmdir /s /q %SOURCEDIR%\examples > /NUL 2>&1
+rmdir /s /q %SOURCEDIR%\api\meshing\datamodel > /NUL 2>&1
+rmdir /s /q %SOURCEDIR%\api\meshing\tui > /NUL 2>&1
+rmdir /s /q %SOURCEDIR%\api\solver\datamodel > /NUL 2>&1
+rmdir /s /q %SOURCEDIR%\api\solver\tui > /NUL 2>&1
```

Comment:
`.\make clean` was not removing `datamodel/*` and `tui/*` .rst files. Now we can remove them using `.\make clean`.